clients/mod/assign/feedback/onlinejudge/backup/moodle2/backup_assignment_onlinejudge_subplugin.class.php

```diff
@@ -27,12 +28,14 @@
  * backup subplugin class that provides the necessary information
  * needed to backup one assignment->onlinejudge subplugin.
  */
-class backup_assignment_onlinejudge_subplugin extends backup_subplugin {
+class backup_assignfeedback_onlinejudge_subplugin extends backup_subplugin
+{
```

Comment:
[Moodle coding style](https://docs.moodle.org/dev/Coding_style) suggests no single line "{". We'd best to follow it.

Reply:
Whole files are now updated with the above coding style.